### PR TITLE
Replace github.com/pkg/errors by stdlib errors

### PR DESCRIPTION
**Pull request by dominikschulz**
RELEASE_NOTES=n/a

Signed-off-by: Dominik Schulz <dominik.schulz@gauner.org>

### CODE REVIEW

File: internal/store/root/init.go [hunk 1]
```diff
@@ -2,22 +2,21 @@ package root
...
-	"github.com/pkg/errors"
 )
 
 // IsInitialized checks on disk if .gpg-id was generated and thus returns true.
 func (r *Store) IsInitialized(ctx context.Context) (bool, error) {
 	if r.store == nil {
 		debug.Log("initializing store and possible sub-stores")
 		if err := r.initialize(ctx); err != nil {
-			return false, errors.Wrapf(err, "failed to initialized stores: %s", err)
+			return false, fmt.Errorf("failed to initialized stores: %w", err)
```
Comment (AnomalRoil) [inline internal/store/root/init.go:19]: typo: "failed to initialize stores: %w"

**Review by AnomalRoil — APPROVED** [review]
Really good idea, I was thinking about it when I coded the key-values code and saw we where using `errors` where `fmt.Errorf` was enough.

Also we might want to once have a discussion about third party dependencies and security. Maybe we should be using Dependabot too, wdyt?


Why the changed images at the beginning without any change to a doc file? Were these uncommitted changes from another commit or are they meant to be here?

**Comment by dominikschulz** [comment]
I'd like to reduce the use of (unnecessary) external deps. Not sure about dependabot. What does it do?

The images shouldn't be changed here. Will revert them.

**Comment by AnomalRoil** [comment]
Dependabot would submit automatically PRs when it detects that one of our dependencies as been upgraded, it allows us to have a clear trace of the dependencies updates and might later allow us to dig further into third party dep security/hardening, since supply chain attacks are becoming more and more of a concern lately in the field.

**Comment by dominikschulz** [comment]
Sounds great.
Could you help to set it up?

**Comment by AnomalRoil** [comment]
Yup, I'll dig into it in its own issue then

**Pull request merged**